sitri/providers/base/base_config_path_mode.py

```diff
@@ -15,7 +15,7 @@ class PathModeStateProvider(ABC):
     _default_path_mode_state: bool = False
 
     def _get_path_mode_state(self, path_mode_param: typing.Optional[bool]) -> bool:
-        return path_mode_param if path_mode_param is not None else self._default_path_mode_state
+        return path_mode_param or self._default_path_mode_state
```

Comment:
Incorrect statement. If user passed path_mode arg we should return path_mode not default.
Example: False or None -> None